### PR TITLE
Update installation documentation for Helm 3.7 OCI

### PR DESCRIPTION
**Pull request by RedbackThomson**
Issue #, if available: https://github.com/aws-controllers-k8s/community/issues/969

Description of changes:
Update any reference to Helm in the installation documentation, removing references to the deprecated OCI chart Helm CLI verbs

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by vijtrip2** [comment]
/approve

Waiting for Jay to review before merging these changes.

Should we add in FAQ section, the error that customers may see when using helm version less than 3.7 ?

**Comment by RedbackThomson** [comment]
> Should we add in FAQ section, the error that customers may see when using helm version less than 3.7 ?

Ah yes! Will add that!

File: docs/content/docs/community/faq.md
Comment (vijtrip2) [inline docs/content/docs/community/faq.md:73]: very minor nit:

```suggestion
[Helm 3.7](https://github.com/helm/helm/releases/tag/v3.7.0) included backward compatibility breaking changes to the manifest format of Helm charts stored in OCI chart repositories. Any images built using Helm <3.7 are not compatible with the latest version of the Helm CLI. This can be solved by using latest version of the chart.
Also remember to use Helm with version 3.7 or above with latest version of chart.
```

**Review by vijtrip2 — APPROVED** [review]

**Pull request merged**
